### PR TITLE
feat: time interval for update pools endpoint

### PR DESCRIPTION
**Pull request by Matthews3301**
In order for `POST $ENDPOINT_URL/pools/x/update` endpoint not to have to be triggered manually, added a time interval env variable which triggers the update lambda at a set interval.

### CODE REVIEW

**Review by garethfuller — COMMENTED** [review]
So does this mean the updates are triggered once per minute for all networks now, not just for Gnosis Chain?

**Comment by Matthews3301** [comment]
@garethfuller yes, that's right.

File: README.md [hunk 1]
```diff
@@ -284,6 +284,7 @@ You can customize your deployment with env variables. See .env.example for all p
 
 #### Capacity Related
 
+- UPDATE_POOLS_INTERVAL_IN_MINUTES - default: 1 - How frequently to run the update pools lambda.
```
Comment (timjrobinson) [inline README.md:287]: I think it should default to 5 minutes as in dev accounts you don't need such frequent updates.

**Review by timjrobinson — APPROVED** [review]
Should default to 5 min for dev mode (keep it 1 min in github workflow for prod). Otherwise LGTM.

**Pull request merged**